src/supermarket/app/views/layouts/application.html.erb

```diff
@@ -43,7 +43,9 @@
             <li><%= link_to 'Learn Chef', learn_chef_url, target: '_blank' %></li>
             <li><%= link_to 'Docs', chef_docs_url, target: '_blank' %></li>
             <li><%= link_to 'Training', chef_training_url, target: '_blank' %></li>
-            <li><%= link_to 'Botbot IRC Logs', 'https://botbot.me/freenode/chef', target: '_blank' %></li>
+            <li><%= link_to 'Join Chef Commuity Slack', 'http://community-slack.chef.io/', target: '_blank' %></li>
+            <li><%= link_to 'Login to Chef Commuity Slack', 'https://chefcommunity.slack.com', target: '_blank' %></li>
+            <li><%= link_to 'Read Slack Logs', 'https://chefcommunity.slackarchive.io', target: '_blank' %></li>
```

Comment:
Maybe consider making these configurable URLs? I don't know how many private Supermarkets customize the linkage in their menus and I also don't know how many private Supermarkets would want to send their users to the community chat site. (I know _we_ as the community would like that.)

Reply:
Configurable for private Supermarkets seems like the right thing to do.  They may want to link to an internal chat platform in addition to or instead of the community slack.